app/helpers/description_helper.rb

```diff
@@ -231,7 +231,7 @@ def description_title(desc)
       result += " (#{permit})"
     end
 
-    return result
+    return t(result)
```

Comment:
Not sure why, but this the description title not being rendered properly. 
https://www.pivotaltracker.com/n/projects/224629/stories/82351780